projects/angular/components/ui-grid/src/ui-grid.intl.ts

```diff
@@ -102,7 +102,11 @@ export class UiGridIntl implements OnDestroy {
      *
      */
     public descending = 'descending';
-
+    /**
+     * No data row message alternative function.
+     *
+     */
+    public noDataMessageAlternative = (_searchValue?: string, _activeFilters?: number | null) => 'No data is available.';
```

Comment:
Looks a bit wrong to declare these arguments with leading `_`.
Also, shouldn't we have tests for the grid for this new logic?
```suggestion
    public noDataMessageAlternative = (searchValue?: string, activeFilters?: number | null) => `No data is available ${searchValue ? ` for ${searchValue}${activeFilters? ' for applied filters': ''}`}.`;
```